BWEB.h

```diff
@@ -92,6 +93,7 @@ namespace BWEB
 		static Map* BWEBInstance;
 
 	public:
+		Map(BWEM::Map& map = BWEM::Map::Instance());

```

Comment:
I'm not sure relying on that there is a static instance is the way to go here. I'd say that the default argument should be removed.

Reply:
I don't like it either, but I want to ensure 100% compatibility with existing code. I don't know is McRave only user, or somebody else use this lib? Do these people willing to change, so I go safe route. I'm always for having explicit dependencies, like seen from my PRs, But this looks safe default as temporary measure for me. So if everybody would like to remove default, I'm fully support that.